src/app/Controllers/ParameterValidator.cs

```diff
@@ -179,10 +282,101 @@ public static JsonResult MovieId(string movieId, string method, ILogger logger)
                 !int.TryParse(movieId.Substring(2), out int val) ||
                 val <= 0)
             {
-                return GetAndLogBadParam("Invalid Movie ID parameter", method, logger);
+                return GetAndLogInvalidMovieIdParameter(method, logger);
             }
 
             return null;
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
```

Comment:
Seems like you could pass one, maybe two params and encapsulate the rest of the values in the method.

Reply:
woops! good catch. forgot to fill out the tag.